src/Catalog/PackageCatalogItemCreator.cs

```diff
@@ -136,7 +144,8 @@ public sealed class PackageCatalogItemCreator : IPackageCatalogItemCreator
                             packageItem.PublishedDate,
                             licenseNames: null,
                             licenseReportUrl: null,
-                            packageHash: packageHash);
+                            packageHash: packageHash,
+                            deprecationItem: deprecationItem);
```

Comment:
Can we not just pass `packageItem.DeprecationInfo` here directly? Why do we need to clone the object?

Reply:
No longer needed indeed. Fixed in https://github.com/NuGet/NuGet.Services.Metadata/pull/542/commits/c9193c2f337d98a07658c107296d03b815243964